docker/build/bin/km-build

```diff
@@ -8,14 +12,14 @@ set -vx
 GOPKG=github.com/mesosphere/kubernetes-mesos
 
 test ${#} -eq 0 && CMD=( make bootstrap install ) || CMD=( "${@}" )
-test -n "$GOPATH" || GOPATH=/pkg
+test -n "$GOPATH" || GOPATH=/go
```

Comment:
I think /pkg was something that I cooked up at 3am one morning. /go and ~/go make more sense.